mod_reforged/hooks/config/strings.nut

```diff
@@ -2019,12 +2028,12 @@ foreach (vanillaDesc in vanillaDescriptions)
 	}),
 	RF_SwordmasterBladeDancer = ::UPD.getDescription({
 		Fluff = "Let's Dance!",
-		Requirement = "Sword"
+		Requirement = "Non-Fencing Sword"
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Gain additional [Initiative|Concept.Initiative] equal to the armor ignore percentage of your equipped sword.",
-				"When using a non-fencing sword, the [Action Point|Concept.ActionPoints] costs of non-AOE skills is reduced by " + ::MSU.Text.colorPositive(1) + " and these skills build up " + ::MSU.Text.colorPositive("25%") + " less [Fatigue.|Concept.Fatigue]",
+				"Gain [Initiative|Concept.Initiative] equal to the armor ignore percentage of your equipped sword.",
```

Comment:
I prefer keeping the `additional` here because otherwise people will get confused if the initiative is a percentage based bonus or a flat increase.